### DISCUSSION TITLE
Projects (beta): Field values should be entities not just strings

### DISCUSSION THREAD

**Discussion by xavierraffin** [community/community#5892] [Projects and Issues] [answered]
If I create custom field on the new project beta. The values that a field take are just strings.

## UX Problem description

If I rename my field "Development" to "development" or "dev team" then I need to rename all my issues one by one which is a pain.
Renaming 200 values by hand is such a pain.

## Possible solution

Instead what I would love is to have like for labels a possibility to edit and rename fields values only once and see the name changing in all issue.

**Comment by Katsute** [comment, marked as answer]
Use a single select

![gh](https://user-images.githubusercontent.com/58778985/135365352-c44f8ce1-b126-4a34-b4ce-affa65a4c940.PNG)

**Reply by xavierraffin** [reply]
thanks that is exactly what I was looking for
